### PR TITLE
Articles

### PR DESCRIPTION
**Pull request by GretaTimaite**

### CODE REVIEW

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Need to get the tests passing.

**Comment by GretaTimaite** [comment]
A quick question: I've downgraded `roxygen2` from version = 7.1.2 to version = 7.1.1 as it was giving me a warning when running checks. Should I have kept the newest version?

Now there's only one warning regarding R version; not sure if I should be concerned about it and/or maybe somehow specify it in the DESCRIPTION file?
` NB: this package now depends on R (>= 3.5.0)
     WARNING: Added dependency on R >= 3.5.0 because serialized objects in
     serialize/load version 3 cannot be read in older versions of R.`

**Review by Robinlovelace — APPROVED** [review]
Happy for this to be merged great work on getting actions working (except on pesky Mac OS) :rocket:

File: vignettes/osmintro.Rmd [hunk 1]
```diff
@@ -371,13 +367,9 @@ You have learned how to download the desired dataset but, most likely, you do no
 saveRDS(leeds, # name of an object in R
```
[the remark this answers is not in the document]
Comment (GretaTimaite) [reply vignettes/osmintro.Rmd:328]: Hm, maybe. I will have a look at it tomorrow. But if I use `download.file()`, doesn't it kind of defeat the purpose of loading the RDS file from one's directory?

[the remark this answers is not in the document]
Comment (Robinlovelace) [reply vignettes/osmintro.Rmd:328]: Depends what the purpose is.

**Comment by Robinlovelace** [comment]
Now it's the Windows ones that are failing :facepalm: 

I suggest you merge this now @GretaTimaite

**Pull request merged**